tasks/apt.yml

```diff
@@ -40,6 +41,8 @@
     - python3-pip
     - python3-dev
     - python-virtualenv
+    - python3-pyparsing
+    - python3-software-properties
```

Comment:
Are you sure these two packages are present on all the supported distributions? I think it would be better to install `python3-pyparsing` and `python3-software-properties` only if the current distribution is debian stretch.